src/pivot-table/components/cells/DimensionValue/StickyCellContainer.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { type ReactNode } from "react";
+import { CELL_PADDING } from "../../shared-styles";
+
+type Props = {
+  isLeftColumn: boolean;
+  children: ReactNode;
+};
+
+const StickyCellContainer = ({ children, isLeftColumn }: Props): JSX.Element => (
+  <div
+    style={{
+      display: "flex",
+      flexDirection: "row",
+      alignItems: "center",
+      gap: `${CELL_PADDING}px`,
+      width: "fit-content",
+      maxWidth: "100%",
+      position: "sticky",
+      left: CELL_PADDING,
+      top: CELL_PADDING,
+      right: isLeftColumn ? undefined : CELL_PADDING,
```

Comment:
```suggestion
      right: !isLeftColumn && CELL_PADDING,
```

Reply:
I honestly prefer using tenary here. I find that to be easier to read and if there is a need to replace `undefined` with another value. There is no need to rewrite the logic.